test/spec/general.js

```diff
@@ -1,9 +1,9 @@
-/* globals SDK_VERSION */
 define(function(require) {
   var OktaAuth = require('OktaAuth');
   var Q = require('q');
   var util = require('../util/util');
   var _ = require('lodash');
+  var packageJson = require('../../package.json');
```

Comment:
Can we load config.json here (seems like we're running the writeconfig task before test as well)?